### PR TITLE
chore!: Update default E2EE settings behavior

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Proposed changes (including videos or screenshots)

We introduced a new E2EE settings with 6.x release - `E2E_Allow_Unencrypted_Messages`. Earlier their default value was kept in a way to not change the default behavior.

We decided to change this default behavior in the major 7.0.0 release and now by default users shouldn't be allowed to send un-encrypted messages in encrypted rooms.
## Further comments

[E2EE2-69](https://rocketchat.atlassian.net/browse/E2EE2-69)

[E2EE2-69]: https://rocketchat.atlassian.net/browse/E2EE2-69?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hugocostadev — CHANGES_REQUESTED** [review]

**Review by MarcosSpessatto — DISMISSED** [review]
The suggestion from @hugocostadev for the changeset is valid @yash-rajpal, I would recommend the same as he did. Can we rewrite that?

**Review by hugocostadev — APPROVED** [review]

File: apps/meteor/server/startup/migrations/v314.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	version: 314,
+	name: 'Update default behavior of E2EE settings, to not allow un-encrypted messages by default and allow files encryption by default.',
+	async up() {
+		await Settings.updateOne(
+			{
+				_id: 'E2E_Allow_Unencrypted_Messages',
+			},
+			{
+				$set: {
+					value: false,
```
Comment (KevLehman) [inline apps/meteor/server/startup/migrations/v314.ts:15]: Why not changing just the packageValue? So if someone has it as `true` and likes it being `true`, we don't break his workflow

**Review by KevLehman — CHANGES_REQUESTED** [review]
Did we change the setting creation code in another PR? The `this.add(...` cause otherwise the code for new workspaces will retain the current behavior as migrations won't run on clean envs

File: apps/meteor/server/startup/migrations/v314.ts (hunk 1 above)
Comment (KevLehman) [reply apps/meteor/server/startup/migrations/v314.ts:15]: This should be said to the customer support team too, so if they receive comments from customers they know what to do 👀

**Review by KevLehman — APPROVED** [review]

**Review by MarcosSpessatto — APPROVED** [review]

**Pull request merged**